Updated "tips" page with CLIENT_MUTE advice. (#1720)
meshtastic/meshtastic: docs/configuration/tips.mdx

## Roles

It is strongly recommended to keep your [ROLE](/docs/configuration/radio/device#roles) set to `CLIENT` or `CLIENT_MUTE`. Only use other roles if you have a specific, well-understood reason to do so. Read our [blog post about choosing the best role](/blog/choosing-the-right-device-role/).

> ### Recommended roles:

Reviewer: ```suggestion
### Recommended Roles
```

> ### Recommended rols: `CLIENT` and `CLIENT_MUTE`

Reviewer: ```suggestion
### Recommended Roles
```

Author: 🤦

> ### Recommended roles: `CLIENT` and `CLIENT_MUTE`

Reviewer: Sorry to be a stickler here but we really do try to keep the headings short for the TOC. Can you please adjust this to just be `Recommended Roles`?

## Roles

> It is strongly recommended to keep your [ROLE](/docs/configuration/radio/device#roles) set to `CLIENT` or `CLIENT_MUTE`. Only use other roles if you have a specific, well-understood reason to do so.

Reviewer: ```suggestion
It is strongly recommended to keep your [ROLE](/docs/configuration/radio/device#roles) set to `CLIENT` or `CLIENT_MUTE`. Only use other roles if you have a specific, well-understood reason to do so. Read our [blog post about choosing the best role](/blog/choosing-the-right-device-role/).
```

:::tip

> For details on choosing a role, read [this blog post](/blog/choosing-the-right-device-role/).

Reviewer: We're trying to steer away from using admonitions(e.g. `tip`) anymore. I think a good alternative to this would be to add an additional sentence to the paragraph above this.

`Read our [blog post about choosing the best role](/blog/choosing-the-right-device-role/).`

Author: Fixed this based on your suggestion.

:::

> ### `CLIENT` and `CLIENT_MUTE` are Recommended

Reviewer: This heading is a bit long, let's go for `Recommended Roles` instead 🙏 it will keep it shorter for the table of contents

Author: Fixed.

- `CLIENT` nodes efficiently repeat and route packets as needed. 
  - **Almost always the correct mode.**

>   - They use smart delays for rebroadcasting, improving network stability.

Reviewer: ```suggestion
  - Uses smart delays for rebroadcasting, improving network stability.
```

  - Use for "rooftop" or other nodes that enhance your mesh.
  - Use during isolated activities such as hiking, skiing, or MTB where you're in an area with few nodes.

> - `CLIENT_MUTE` nodes behave as above but do not repeat packets:

Reviewer: ```suggestion
- `CLIENT_MUTE` nodes behave as above but do not repeat packets.
```

>   - Personal nodes that are in range of a higher-profile node in a dense or congested mesh.

Reviewer: ```suggestion
  - Use for personal nodes that are in range of a higher-profile node in a dense or congested mesh.
```

>   - Enthusiasts with multiple nodes (set your "best-placed" node to `CLIENT`)

Reviewer: ```suggestion
  - Use when you have multiple nodes in close proximity (set your "best-placed" node to `CLIENT`).
```

![Router Node Example](/img/configuration/router.webp)
*One example of a 'Router' node.  Photo credit: Cully@KBOXLABS*

> 

Reviewer: Shouldn't need to add more new lines

Author: Fixed, good catch.
